include/rdma/fabric.h

```diff
@@ -121,6 +121,7 @@ typedef struct fid *fid_t;
 #define FI_TRANSMIT_COMPLETE	(1ULL << 27)
 #define FI_DELIVERY_COMPLETE	(1ULL << 28)
 
+#define FI_SELECTIVE_COMPLETION	(1ULL << 55)
```

Comment:
I'm clearly very confused by this grouping/numbering scheme.  59 is already in use by `FI_DIRECTED_RECV` and 60 is reserved for provider-specific values.  `FI_SEND` is passed into `fi_bind` in the same bind flags argument as `FI_SELECTIVE_COMPLETION` but it has a value of 13.  I can make the change that you are requesting, but I'm baffled at how I would arrive at that decision on my own...